src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js

```diff
@@ -52,6 +52,223 @@ xdescribe('Discharge Upgrade Wizard', () => {
 
       // PREVIOUS_APPLICATION
       h.verifyUrl(ROUTES.PREV_APPLICATION);
+      h.selectRadio(h.PREV_APPLICATION_INPUT, 0);
```

Comment:
Flagging (ignore if this is already done / irrelevant, etc.) that we'll have to test quite a few flows, and you may want to start breaking things down into multiple folders / files.

Reply:
Thanks Randi! Yes I am definitely working on that, I will be testing all branched flows and skip ahead logic